presto-main/src/test/java/com/facebook/presto/cost/TestJoinStatsRule.java

```diff
@@ -90,6 +95,21 @@
             NORMALIZER,
             1.0);
 
+    private StatsCalculatorTester defaultJoinSelectivityEnabledTester;
+
+    @BeforeClass
+    public void setupClass()
+    {
+        defaultJoinSelectivityEnabledTester = new StatsCalculatorTester(getTestSession());
+    }
+
+    @AfterClass(alwaysRun = true)
+    @Override
+    public void tearDown()
```

Comment:
I think you need to rename this something that won't conflict with the teardown method from the superclass